dbld/images/debian-bullseye.dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM debian:bullseye
+LABEL maintainer="Andras Mitzki <andras.mitzki@balabit.com>, Laszlo Szemere <laszlo.szemere@balabit.com>, Balazs Scheidler <balazs.scheidler@oneidentity.com>"
+ENV OS_DISTRIBUTION=debian
+ENV OS_DISTRIBUTION_CODE_NAME=bullseye
+
+ARG ARG_IMAGE_PLATFORM
+ARG COMMIT
+ENV IMAGE_PLATFORM ${ARG_IMAGE_PLATFORM}
+LABEL COMMIT=${COMMIT}
+
+ENV DEBIAN_FRONTEND=noninteractive
+ENV DEBCONF_NONINTERACTIVE_SEEN=true
+ENV LANG C.UTF-8
+
+COPY images/fake-sudo.sh /usr/bin/sudo
+COPY images/entrypoint.sh /
+COPY . /dbld/
+
+RUN /dbld/builddeps install_dbld_dependencies
+RUN /dbld/builddeps install_apt_packages
+RUN /dbld/builddeps install_debian_build_deps
+RUN /dbld/builddeps install_criterion
+
```

Comment:
`RUN /dbld/builddeps install_bison_from_source` is missing from here. See `Debian 11` at https://repology.org/project/bison/versions

Reply:
Bison is an optional dependency as we build from tarball, right?

Reply:
Yes, you are right, we do not install bison on any other of our "simple" builder platforms. The kira image has it, but it is justified.